### PR TITLE
created-html-css-js-setup

### PR DESCRIPTION
**Pull request by ghost**
# How to test
- Look at the code and make sure it has all the right elements and is correctly linked to main.js and main.cs.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+console.log('frogs r radd');
```
Comment (ghost) [inline main.js]: I see a console log 🙈

**Comment by AlexOrga** [comment]
Can we move the script tag to before the closing body tag instead of in the head?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>The Frog Sayeth Croak</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
+    <script src="main.js"></script>
+</head>
+<body>
+    <div class="container"></div>
+        <div id="home-container"></div>
+
```
Comment (ghost) [inline index.html]: we need a script tag to link main.js

File: main.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+function printToDom(stuffToPrint, divId) {
```
Comment (ghost) [inline main.js]: This will be good for all of us to have, but going forward, it would be better to save this until we have something to pass to it.

Comment (ghost) [reply main.js]: Taking it out of this ticket. It is in a different ticket so we'll write it later.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>The Frog Sayeth Croak</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
+    <script src="main.js"></script>
```
Comment (ghost) [inline index.html]: The script tag needs to move to bottom of the body

File: index.html [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+</head>
+<body>
+    <div class="container"></div>
+        <div id="home-container"></div>
```
Comment (ghost) [inline index.html]: Do we need more than one container on the home page?

Comment (ghost) [reply index.html]: We do not! I took it out.

File: main.js (hunk 1 above)
Comment (ghost) [reply main.js]: Done.

File: index.html (hunk 2 above)
Comment (ghost) [reply index.html]: Okay, fine.

File: index.html (hunk 4 above)
Comment (ghost) [reply index.html]: Done.

File: index.html [hunk 6]
```diff
@@ -6,11 +6,12 @@
     <title>The Frog Sayeth Croak</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
-    <script src="main.js"></script>
 </head>
 <body>
     <div class="container"></div>
-        <div id="home-container"></div>
-    
+        <div id="home"></div>
```
Comment (ghost) [inline index.html]: We don't need a separate div here. You can add the id after the class on the first one.
`<div class="container" id="home">`

Comment (ghost) [reply index.html]: YOU ARE RIGHT, GIRRL!

**Comment by ghost** [comment]
👍

**Comment by ledugani** [comment]
🏀

**Comment by AlexOrga** [comment]
im all thumbs

**Comment by Cgatewood55** [comment]
:+1:

**Pull request merged**